app/src/main/java/com/github/se/eventradar/ui/hosting/Hosting.kt

```diff
@@ -102,6 +112,22 @@ fun HostingScreen(
         placeholderStringResource = R.string.hosting_search_placeholder)
 
     val events = getEventsBasedOffUiState(uiState)
+    var mapTopConstraint = searchfilter.bottom
```

Comment:
this indeed the proper use of constraind layout childs - seems all good to me, only improvement I can suggets but do not insist on is doing as done for HomeScreen etc where the items displayed are actually composable functions in ReusableComponents. so for excample you would have composable function in Reusabele components called something along the lines of TextFilterButNoMatch(modifier: Modifier) { } and then you would just call it at line 119 - makes eveything more clean and organised. should take u 8 mins, no more

Reply:
I made these changes and it only neatens it by two lines, since the modifier has to be passed in still. Therefore, I have elected not to make these changes.